### ISSUE TITLE
Show default per-kind icons for all entries in Component Browser.

### ISSUE THREAD

**Issue by wdanilo** [enso-org/enso#4451]
This task is automatically imported from the old Task Issue Board and it was originally created by Mateusz Czapliński.
Original issue is [here](https://www.pivotaltracker.com/story/show/182584326).
<hr />

## Why

**As** Enso User
**I want** to see default per-kind icons in Component Browser
**So that** I can quickly visually distinguish the kind of any item in the Component Browser.

## Acceptance Criteria

```gherkin
Scenario: 
GIVEN a newly created project
WHEN I open the Component Browser
AND I scroll through all of its groups and sections
THEN every entry representing a component has an icon depending on its kind
AND THEN if I check the kind of the item in the Standard Library code, the kind matches the icon.
```

## Blockers:
[#182584322](https://www.pivotaltracker.com/story/show/#182584322) resolved
[#181869402](https://www.pivotaltracker.com/story/show/#181869402) resolved

## Comments:
**Mateusz Czapliński** reports a new **STANDUP** for yesterday (2022-07-05):

**Progress:** Did quick initial Technical Design discussion with AO. Started looking into the code on AO's Component Browser integration branch (required for completing the task). It should be finished by 2022-07-11.

**Next Day:** Next day I will be working on the [#same task](https://www.pivotaltracker.com/story/show/182584326). Implement and test. Discuss with AO how to proceed with regard to the required dependency on the Component Browser integration branch. Open a PR if possible. (Enso Bot - Jul 6, 2022)
<hr />
**Mateusz Czapliński** reports a new **🔴 BLOCKER** for today (2022-07-06):

**Summary:** The [Show default per-kind icons for all entries in Component Browser. (#182584326) task](https://www.pivotaltracker.com/story/show/182584326) is blocked by the [#181869402 task](https://www.pivotaltracker.com/story/show/181869402).

Task #181869402 introduces Component Browser integration features on which #182584326 is built. The initial implementation of #182584326 is done and tested, but opening a PR & comprehensive testing is not possible due to the integration branch being not in final state and not merged into `develop`. The PR for the integration task (#181869402) is expected today suggesting a Blocker Size of 1 day, but based on that task's complexity and noticing it had delays in the past, I expect that the delay (Blocker Size) may increase. In meantime, after consulting with AO, I'm planning to work first on debugging and fixing a colors issue in #180892146. When done, will further consult with AO if still blocked. (Enso Bot - Jul 6, 2022)
<hr />
**Mateusz Czapliński** reports a new **🔴 DELAY** for today (2022-07-14):

**Summary:** There is **7 days** delay in implementation of the [Show default per-kind icons for all entries in Component Browser. (#182584326) task](https://www.pivotaltracker.com/story/show/182584326).
It **will cause 0 days delay** for the delivery of this weekly plan.

**Delay Cause:** 4 days of delay due to being blocked by #181869402. Then, 3 days of delay due to weekend/vacation. (Enso Bot - Jul 14, 2022)
<hr />
**Mateusz Czapliński** reports a new **STANDUP** for today (2022-07-14):

**Progress:** Updated to new `develop`. Re-tested and opened a PR. It should be finished by 2022-07-18.

**Next Day:** Next day I will be working on the [##182713626 task](https://www.pivotaltracker.com/story/show/182713626). Start looking at the bug. Discuss with AO to verify if the expected cause of the bug is bad layer of the label. (Enso Bot - Jul 14, 2022)
<hr />
**Mateusz Czapliński** reports a new **🔴 DELAY** for yesterday (2022-07-19):

**Summary:** There is **1 day** delay in implementation of the [Show default per-kind icons for all entries in Component Browser. (#182584326) task](https://www.pivotaltracker.com/story/show/182584326).
It **will cause 0 days delay** for the delivery of this weekly plan.

**Delay Cause:** Late code-review + addressing code-review comments. (Enso Bot - Jul 20, 2022)
<hr />
**Mateusz Czapliński** reports a new **STANDUP** for yesterday (2022-07-19):

**Progress:** Addressed code-review comment. It should be finished by 2022-07-19. (Enso Bot - Jul 20, 2022)
<hr />
**Acceptance/QA** review set to **pass** (Adam Obuchowicz - Jul 20, 2022)
<hr />
